lib/utils/logUtil.ts

```diff
@@ -2,13 +2,23 @@ import winston, { format, Logger } from "winston";
 
 const PROCESSOR_NAME = "ldes-client";
 
-const consoleTransport = new winston.transports.Console();
+const consoleTransport = new winston.transports.Console({
+    stderrLevels: [
```

Comment:
Any reason you didn't include emerg, alert, crit, warning, notice besides that we are not using them anyways (but beware our future selves)?

Reply:
Added in https://github.com/rdf-connect/ldes-client/pull/94/commits/a8b2e194ac49d22c7b080c4a3ce4b5dc7c6434a3